src/md/compiler/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(MDCOMPILER_SOURCES
   import.cpp
   importhelper.cpp
   mdutil.cpp
+  newmerger.cpp
```

Comment:
You're fast; I hadn't even hit "enter" to tag you yet...

I had to add newmerger.cpp to fix a build break due to the following in regmeta.h in the RegMeta class:

```
#ifdef FEATURE_METADATA_EMIT_ALL
    NEWMERGER   m_newMerger;                // class for handling merge
#endif //FEATURE_METADATA_EMIT_ALL
```